dev-java/ant_core/ant_core-1.10.7.recipe

```diff
@@ -54,7 +54,7 @@ BUILD()
 {
 	source /system/data/profile.d/openjdk8.sh
 	export PATH=$JDK8_HOME/bin:$PATH
-	ANT_HOME= ./build.sh
+	ANT_HOME=./build.sh
```

Comment:
This line seems nonsensical. It basically makes the `BUILD` phase do nothing; the compilation then happens on the `INSTALL` phase.